primitives/src/merkle_tree/field_based_mht/ramht/mod.rs

```diff
@@ -0,0 +1,75 @@
+use algebra::Field;
+use std::fmt::Debug;
+
+pub mod poseidon;
+
+//TODO: We can just keep this and remove the implementation at field_based_mht/mod.rs ?
```

Comment:
Probably we can get rid of the implementation at field_based_mht/mod.rs and just keep this one